6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<ul><li>Add periods for each sentence at the end of your bullet points.</li><li>I don't really know what "This is the required posterior" means -- elaborate a bit more?</li></ul>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
- Added periods.
- Replaced "This is the required posterior" with "These sampled values are an approximation of our model's posterior."